example.ts

```diff
@@ -22,12 +23,16 @@ let alternativeGet = await twitterApi.request("GET", "statuses/user_timeline.jso
   tweet_mode: "extended", // Used to prevent truncating tweets
 });
 
-// These post requests do not work in the current state and are here
-// to show how someone would be able to use the post function
```

Comment:
This comment is needed because the following examples do not actually work and are only here to show the structure for post requests.